Cargo.toml

```diff
@@ -13,20 +13,16 @@ categories = ["no-std", "embedded"]
 exclude = [ "demo/*" ]
 
 [dependencies]
+fixed-hash = { version = "0.3.0-beta", default-features = false }
+uint = { version = "0.4", default-features = false }
+byteorder = { version = "1", default-features = false }
 pwasm-alloc = { path = "alloc", version = "0.4" }
 pwasm-libc = { path = "libc", version = "0.2" }
 tiny-keccak = "1"
-
-[dependencies.byteorder]
-version = "1"
-default-features = false
-
-[dependencies.parity-hash]
-version = "1"
-default-features = false
+static_assertions = "0.2"
 
 [features]
-default = []
-std = ["parity-hash/std"]
+default = ["std"]
```

Comment:
it should not be default

pwasm-std is designed to replace std, not to come with default std, so this feature is activated only in some testing scenarios

Reply:
Yeah, but many other crates don’t aim to replace std by default :) so yeah. I’m against

Reply:
okay will revert this then

Reply:
Do you have an idea how we could solve the problems with doc-tests for this crate?

Reply:
> okay will revert this then

done